Cabal/Distribution/PackageDescription/Configuration.hs

```diff
@@ -189,7 +190,7 @@ instance Monoid d => Mon.Monoid (DepTestRslt d) where
     mappend (MissingDeps d) (MissingDeps d') = MissingDeps (d `mappend` d')
 
 
-data BT a = BTN a | BTB (BT a) (BT a)  -- very simple binary tree
+data Tree a = Tree a [Tree a]
```

Comment:
this is now exactly the same as http://hackage.haskell.org/package/containers-0.5.6.3/docs/Data-Tree.html